assets/javascripts/builder.jsx

```diff
@@ -0,0 +1,35 @@
+var React = require('react');
+var riek = require('riek')
+var RIEInput = riek.RIEInput;
+var FormElement = require('./formElement.jsx');
+
+module.exports = React.createClass({
+  render: function() {
+    var formElements = [];
+    var body;
+
+    for (var i = 0; i < this.props.formElements.length; i++) {
```

Comment:
It's better to declare the `var i` at the top, then just `for (i = 0...` for the loop.